CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ ___
 ## Version 1.7.1
 
 ### Bugfixes
-- Fix AppCenterReactNativeShared pod version issue for appcenter-crashes module. #361
+- Fix AppCenterReactNativeShared pod version issue for appcenter-crashes module. https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/361
```

Comment:
Let's use `[#361](https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/361)`

Reply:
I agree, let's use the pretty formatting.  I had started doing that in some of the 1DS docs and we should do it here too